c2corg_ui/static/js/associationcarddirective.js

```diff
@@ -0,0 +1,17 @@
+goog.provide('app.associationCardDirective');
+
+goog.require('app.utils');
+
+
+/**
+ * @return {angular.Directive} Directive Definition Object.
+ */
+app.associationCardDirective = function() {
+  var url = app.utils.buildDocumentUrl('{{doc.documentType}}', '{{doc.document_id}}', '{{doc.locales[0].lang}}');
+  return {
+    restrict: 'E',
+    template: '<a ng-href="' + url + '">{{doc.locales[0].title}}</a>'
```

Comment:
`{{doc.locales[0].title}}` is not going to change, right? You could use an one-time-binding: `{{::doc.locales[0].title}}`